pkg/scene/widget_area.go

```diff
@@ -0,0 +1,68 @@
+package scene
+
+import "github.com/reearth/reearth-backend/pkg/id"
+
+// WidgetArea has the widgets and alignment information found in each part area of a section.
+type WidgetArea struct {
+	widgetIds []id.WidgetID
+	align     string
+}
+
+var (
+	WidgetAreaTop    = "top"
+	WidgetAreaMiddle = "middle"
+	WidgetAreaBottom = "bottom"
+)
```

Comment:
I tried this but run into issues in plugin/manifest/convert.go when I am converting between packages. Is the above suggested completely necessary? It becomes so that I can't convert values to another library because I can't access a primitive value (string, etc). 

Is there a way to do that? I searched for newType pattern and it looks like it is a Rust thing, and can't find anything on naming constants in Golang in this way. I also looked at some of the packages in the standard library and couldn't find any constants using this pattern.

Please let me know if I can ignore this suggestion or if you know how to convert between packages while using the newtype pattern.

Reply:
I mean newtype pattern is just `type WidgetAreaType string`. I just looked them up and found out that they are called "defined types" and "named types".

They can be easily converted into a string:

```go
string(scene.WidgetAreaTop)
```

Reply:
Gotcha, thanks!